docs/config.md

```diff
@@ -328,24 +328,16 @@ Can be customized by the `format_short_signature()` template alias.
 
 ### Allow "large" revsets by default
 
-Certain commands (such as `jj rebase`) can take multiple revset arguments, and
-each of these may resolve to one-or-many revisions. By default, `jj` will not
-allow revsets that resolve to more than one revision &mdash; a so-called "large
-revset" &mdash; and will ask you to confirm that you want to proceed by
-prefixing it with the `all:` modifier.
+By default, `jj` requires the `all:` prefix for many commands to accept a revset
```

Comment:
nit: "many commands" seems wrong. Maybe reuse the original explanation?

> Certain commands (such as `jj rebase`) can take multiple revset arguments, but require the `all:` prefix ...

Reply:
Good point, thanks! I also needed to replace "prefix" with "modifier" here.

Let me know if you think this needs more changes.